CONTRIBUTING.md

```diff
@@ -62,6 +62,7 @@ your host environment:
 -   [Nix](https://nixos.org/download.html) to provide a reproducible development
     environment.
 -   [Direnv](https://direnv.net/) to load that environment automatically in your shell.
+-   [Poetry](https://python-poetry.org/) to test and lint the code.
```

Comment:
When you follow the instructions, nix and direnv will provide poetry to you, along with any other requirements, so it's not really a core dependency.

Reply:
Oh - that actually did not work for me. When I followed the instructions, the `direnv allow` command failed because poetry was not available. So then I installed it separately, and re-tried `direnv allow` which did create an environment for me then.

Speculation: perhaps it has something to do with the platform I'm on? Nix might not have any poetry versions tagged for an M1 Mac.

Reply:
Hmm maybe... could you post the logs?

Side note: It'd be nice to add that arch to our CI when https://github.com/github/roadmap/issues/528 is fixed.

Reply:
Sure, this is what I see when poetry is not available:
```
$ direnv allow

direnv: loading ~/Documents/GitHub/copier/.envrc
direnv: using flake . --impure --accept-flake-config
error: experimental Nix feature 'flakes' is disabled; use '--extra-experimental-features flakes' to override
direnv: No virtual environment exists. Executing `poetry install` to create one.
/bin/bash:7: poetry: command not found
/bin/bash:8: poetry: command not found
direnv: export +POETRY_ACTIVE +VIRTUAL_ENV -DYLD_LIBRARY_PATH ~PATH
```

Reply:
> error: experimental Nix feature 'flakes' is disabled; use '--extra-experimental-features flakes' to override

This is the problem.

Try installing [this version of direnv](https://github.com/nix-community/nix-direnv#installation) (which automatically enables flakes when needed), or [enabling flakes on your system](https://nixos.wiki/wiki/Flakes#Permanent).

Reply:
Alternatively, you can run `nix-shell` to enter a devshell without needing flakes.

Reply:
It should be fixed with https://github.com/copier-org/copier/commit/bc3609699e473b22a6d9543c26610debb125b180. Please check if you still get the error, and open an issue otherwise.

Thanks!

Reply:
Thank you!